### ISSUE TITLE
Pause button shows up instead of Play while debugging multiple processes

### ISSUE THREAD

**Issue by ramya-rao-a** [Microsoft/vscode#14333]
- VSCode Version: 1.7-insiders
- OS Version: Windows

Testing #14171

Steps to Reproduce:

1. Have the below configurations in `launch.json`
```
"configurations": [
        {
            "name": "Hello",
            "type": "node",
            "request": "launch",
            "program": "${workspaceRoot}/hello/hello.js",
            "console": "internalConsole"
        },
        {
            "name": "Bye",
            "type": "node",
            "request": "launch",
            "program": "${workspaceRoot}/bye/bye.js",          
            "console": "internalConsole"
        },
        {
            "name": "Multi",
            "type": "composite",
            "configurationNames": [
                "Hello",
                "Bye"
            ]
        }
    ]
```

2. Have breakpoints set in both `hello.js` and `bye.js`.
3. Launch the "Multi" configuration
4. Switch between the callstacks of the 2 processes, choose any and hit F5 or the Play button in the debug bar to continue
5. Select the remaining process in the callstack pane. 
6. At this point you would expect the Play button to appear, instead the Pause button appears. The Step Into/Over/Out buttons are disabled

**Comment by isidorn** [comment]
Fixed via db09b30

**Comment by ramya-rao-a** [comment]
@isidorn  thanks!
